<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>通讯录系统</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
	border-image: url(:/images/flowers.png);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>10</pointsize>
       <weight>10</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton
{
	background:rgba(246, 249, 245,0.15);
	font: 87 10pt &quot;Arial Black&quot;;
	color:  rgb(215, 223, 218);
}
</string>
     </property>
     <property name="text">
      <string>查找</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/search.png);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#lineEdit
{
	background:rgba(12,10,20,0.3);
	color: rgb(215, 223, 218);
}</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-6</y>
      <width>981</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget_3
{
	background:rgba(12,10,20,0.4);
}</string>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>551</width>
       <height>591</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">#listWidget
{
	background:rgba(12,10,20,0.3);
}</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>10</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_2
{
	background:rgba(246, 249, 245,0.15);
	font: 87 10pt &quot;Arial Black&quot;;
	color:  rgb(215, 223, 218);
}
</string>
     </property>
     <property name="text">
      <string>新建</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>10</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_3
{
	background:rgba(246, 249, 245,0.15);
	font: 87 10pt &quot;Arial Black&quot;;
	color:  rgb(215, 223, 218);
}
</string>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>54</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>14</pointsize>
       <weight>10</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#label
{
	background:rgba(12,10,20,0.3);
	font: 87 14pt &quot;Arial Black&quot;;
	color:  rgb(215, 223, 218);
}
</string>
     </property>
     <property name="text">
      <string>姓名</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>54</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_2
{
	background:rgba(12,10,20,0.3);
	font: 87 14pt &quot;Arial Black&quot;;
	color:  rgb(215, 223, 218);
}
</string>
     </property>
     <property name="text">
      <string>电话</string>
     </property>
    </widget>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>120</y>
       <width>361</width>
       <height>511</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#listView
{
color: rgb(215, 223, 218);
	background:rgba(12,10,20,0.35);
}</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="autoScrollMargin">
      <number>16</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>110</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#lineEdit_2
{
	background:rgba(12,10,20,0.3);
	color: rgb(215, 223, 218);
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>210</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#lineEdit_3
{
	background:rgba(12,10,20,0.3);
	color: rgb(215, 223, 218);
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>310</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#lineEdit_4
{
	background:rgba(12,10,20,0.3);
	color: rgb(215, 223, 218);
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>420</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#lineEdit_6
{
	background:rgba(12,10,20,0.3);
	color: rgb(215, 223, 218);
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>310</y>
       <width>54</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_3
{
	background:rgba(12,10,20,0.3);
	font: 87 14pt &quot;Arial Black&quot;;
	color:  rgb(215, 223, 218);
}
</string>
     </property>
     <property name="text">
      <string>地址</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>420</y>
       <width>54</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_4
{
	background:rgba(12,10,20,0.3);
	font: 87 14pt &quot;Arial Black&quot;;
	color:  rgb(215, 223, 218);
}
</string>
     </property>
     <property name="text">
      <string>关系</string>
     </property>
    </widget>
    <zorder>pushButton_2</zorder>
    <zorder>listWidget</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>listView</zorder>
    <zorder>lineEdit_2</zorder>
    <zorder>lineEdit_3</zorder>
    <zorder>lineEdit_4</zorder>
    <zorder>lineEdit_6</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
   </widget>
   <zorder>widget_3</zorder>
   <zorder>widget</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
